debian/rules

```diff
@@ -4,5 +4,6 @@
 	dh $@
 
 override_dh_auto_build:
+	make
```

Comment:
`dh` by default runs `make` during its `dh_auto_build` phase and it so happens that we do have a `Makefile` in the repo's root so it was actually doing something.

This [commit](https://github.com/KanoComputing/kano-desktop/commit/ecc23791ad89ed8e244fe33fd77b873e6f9aaa65) introduced the

```make
override_dh_auto_build:
	cd po && make messages.pot
	cd po && make
```

which completely overrides the default behaviour and hence the `Makefile` in the root is never run and hence the `kano-mount-trigger` executable is never compiled and so fails to be installed.

TL;DR The package fails to build without it.